XouDevSpec/Classes/Model/ApiCallerModel.swift

```diff
@@ -88,7 +88,7 @@ public struct ApiCallerModel {
       }
     public  func getMangaData(completetionHandler: @escaping(Result<[MangaDetails], AnimeError>) -> Void) {
     
-        var request = URLRequest(url: NSURL(string:  "https://jikan1.p.rapidapi.com/search/anime?q=\(searchText)")! as URL,
+        var request = URLRequest(url: NSURL(string:"https://jikan1.p.rapidapi.com/search/manga?q=\(searchText)")! as URL,
```

Comment:
for NSURL I would suggest something like this to avoid force unwrapping:
You could use a guard let here as well and use `as?` to convert to URL. 
`guard let nsUrl = NSURL(string....), let url = nsURl as? URL else { 
      send back error using callback
} 
var request = URLRequest(url: url)
`

Reply:
Good point, actually I think you could also directly create a URL right ? Instead of us creating a NSURL from string and then trying to cast it to URL, just create a URL from string directly

Reply:
Yea the whole NSURL stage can be skipped and use the URL init instead.